doc/release-notes/6.1-release-notes.md

```diff
@@ -0,0 +1,195 @@
+
+Please see the "Installing Solr" section of the Installation Prerequisites guide.
+
+
+### Development
+- Developers can enjoy a dramatically faster feedback loop when iterating on code if they are using IntelliJ IDEA Ultimate (free educational licenses are available) and the Payara Platform Tools plugin.
+For details, see http://preview.guides.gdcc.io/en/develop/container/dev-usage.html#intellij-idea-ultimate-and-payara-platform-tools
+
+- There is now a Markdown (.md) previewer: https://dataverse-guide--9986.org.readthedocs.build/en/9986/user/dataset-management.html#file-previews
+
+- A new version of the standard Dataverse Previewers from https://github/com/gdcc/dataverse-previewers is available. The new version supports the use of signedUrls rather than API keys when previewing restricted files (including files in draft dataset versions). Upgrading is highly recommended.
+  - SignedUrls can now be used with PrivateUrl access tokens, which allows PrivateUrl users to view previewers that are configured to use SignedUrls. See #10093.
+  - Launching a dataset-level configuration tool will automatically generate an API token when needed. This is consistent with how other types of tools work. See #10045.
+
+## OpenID Connect Authentication Provider Improvements
+
+### Using MicroProfile Config For Provisioning
+
+With this release it is possible to provision a single OIDC-based authentication provider
+by using MicroProfile Config instead of or in addition to the classic Admin API provisioning.
+
+If you are using an external OIDC provider component as an identity management system and/or broker
+to other authentication providers such as Google, eduGain SAML and so on, this might make your
+life easier during instance setups and reconfiguration. You no longer need to generate the
+necessary JSON file.
+
+### Adding PKCE Support
+
+Some OIDC providers require using PKCE as additional security layer. As of this version, you can enable
+support for this on any OIDC provider you configure. (Note that OAuth2 providers have not been upgraded.)
+
+## Improved Testing
+
+With this release, we add a new type of testing to Dataverse: integration tests which are no end-to-end tests
+like our API tests. Starting with OIDC authentication support, we test regularly on CI for working condition
+of both OIDC login options in UI and API.
+
+The testing and development Keycloak realm has been updated with more users and compatibility with Keycloak 21.
+
+The support for setting JVM options during testing has been improved for developers. You now may add the
+`@JvmSetting` annotation to classes (also inner classes) and reference factory methods for values. This improvement is
+also paving the way to enable manipulating JVM options during end-to-end tests on remote ends.
+
+As part of these testing improvements, the code coverage report file for unit tests has moved from `target/jacoco.exec` to `target/coverage-reports/jacoco-unit.exec`.
```

Comment:
All of the above about testing easily falls under developement if we'd like to move it there.

Reply:
moved